docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md

```diff
@@ -0,0 +1,199 @@
+
+![](imgs/2_mac_install_2.png)
+
+Then type these two lines to start the installation
+
+```json
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
+```
+
+```json
+bash Mambaforge-$(uname)-$(uname -m).sh
+```
+
+You  just follow the instructions, press `Enter` and type _yes_ like here:
+
+![](imgs/2_mac_install_3.png)
+
+When you see this:
+
+![](imgs/2_mac_install_4.png)
+
+you are finished! Close and reopen the Terminal now. Happy coding! 👍
+
+## Using Mambaforge 
+
+Now we use Mambaforge by opening the command line. If you are not familiar with the command line yet, you can check out Roberts tutorial [here](https://www.youtube.com/watch?v=MOEPe9TGBK0&t=1146s). 
+
+To open the Command Prompt in Windows, press the `Windows button`, type _cmd_ and press `Enter`. The Mac OS users should already know how to open the Terminal ;-)
+
+When you open the command line, it should look like this: 
+
+![](imgs/3_envs_1.png)
+
+You are now located in the base environment. It is the default environment and includes beside a Python installation the core Conda libraries and dependencies.
+
+> **_Strong recommendation:_** Never install any packages into the base environment! 
+
+The reason for this is that incompatibilities between packages can occur. Robert demonstrated this [here](https://focalplane.biologists.com/2022/12/08/managing-scientific-python-environments-using-conda-mamba-and-friends/). If this happens in any environment apart from the base environment it is no problem. You can delete the environment, recreate it and start again. If this happens in the base environment, you need to delete and reinstall Mambaforge. 
+
+## Creating a new environment 
+You can create a new environment typing the following command into the Command Prompt: 
+
+```json
+mamba create -n my_first_env devbio-napari python=3.9 -c conda-forge 
+```
+This will create a new environment with the name `my_first_env` and with Python version 3.9 installed. Furthermore, the latest version of devbio-napari will be installed in this environment, too. Devbio-napari is a collection of Python libraries and Napari plugins maintained by the BiAPoL team, that are useful for processing fluorescent microscopy image data.
+Conda will ask you about your permission to download the needed packages with `Proceed [y]/n`. By hitting `Enter` you confirm this and mamba will download and install the necessary packages. 
+
+> **_Recommendation:_** Create one conda environment for every project you are working on. This allows you to keep an overview on the needed packages for the project, maintaining them and ensure compatibility of the packages. 
+
+To activate the environment, type: 
+```json
+mamba activate my_first_env
+```
+This should lead to the prefix my_first_env appearing at the beginning of the command line: 
+
+![](imgs/3_envs_2.png)
+
+## Starting napari 
+
+Now you can open napari, just type:
+```json
```

Comment:
Just a side note, why `json` here? I would say they are `bash` comments. Probably doesn't give you the red coloring, though. SOmething that probably only the codeheads like us notice ^^